src/pages/Dashboard/index.tsx

```diff
@@ -23,9 +22,7 @@ const Dashboard = () => {
           <StatsWrapper isShowWatchList={WATCHLIST_ACCESS[chainId]}>
             <Portfolio />
 
-            {WATCHLIST_ACCESS[chainId] && (
-              <WatchList visibleTradeButton={true} tradeLinkUrl={MENU_LINK.swap} redirect={true} />
-            )}
+            {WATCHLIST_ACCESS[chainId] && <WatchList />}
```

Comment:
Now that the Watchlist isn't Chain dependent, we should actually show the watchlist in every chains Dashboard

Reply:
yes I will make that change